src/value.rs

```diff
@@ -194,16 +194,12 @@ impl _Timestamp {
         }
     }
     #[getter]
+    pub fn get_time(&self) -> PyResult<u64> {
```

Comment:
does it need to be a result? the API will not break even if it needs to become one considering errors are forwarded as exceptions

Reply:
Good point. Addressed.